src/navigation/navigation.tsx

```diff
@@ -395,6 +396,22 @@ const QRStack = (): JSX.Element => {
           header: (props: NativeStackHeaderProps) => <SSIHeaderBar {...props} />,
         }}
       />
+      <Stack.Screen
+        name={ScreenRoutesEnum.LOCK}
+        component={SSILockScreen}
+        initialParams={{onVerificationSuccess: onLogin}}
+        options={({route}) => ({
+          headerTitle: route.params.headerTitle ? route.params.headerTitle : translate('lock_title'),
```

Comment:
since we stepped away from the reusing the same stack, we can set only route.params.headerTitle as title as this screen is only used to enter your pin code

Reply:
i see i copied the wrong thing, i meant to say we can set the title with translate('authenticate_screen_title'). so we can omit the whole route.params.headerTitle

Reply:
changed